lib/kb_uploadmethods/Utils/UnpackFileUtil.py

```diff
@@ -38,14 +39,22 @@ def _file_to_staging(self, file_path_list, subdir_folder=None):
         subdir_folder_str = '/' if not subdir_folder else '/{}'.format(subdir_folder)
         staging_service_host = self._staging_service_host()
         end_point = staging_service_host + '/upload'
-        headers = {'Authorization': self.token}
 
         files = {'destPath': subdir_folder_str}
 
         for file_path in file_path_list:
             files.update({'uploads': (os.path.basename(file_path), open(file_path, 'rb'))})
 
-            resp = _requests.post(end_point, headers=headers, files=files)
+            multipart_encoded = MultipartEncoder(files) # Supports files over ~2gb (`requests` lib maximum)
```

Comment:
do you think we can do try: normal `_requests.post` and in the exception clause we do `multipart_encoded`? The reason is I think we don't have any test coverage for `_file_to_staging` due to the fact test docker images won't have staging access. I think it's safer to do that since `unpack_web_file` is a popular method.

Reply:
Ah, good idea, I'll make that change

Reply:
@dauglyon are you still working on this or has it stalled?